core/lib/multivm/src/tracers/validator/vm_latest/mod.rs

```diff
@@ -81,6 +83,50 @@ impl<H: HistoryMode> ValidationTracer<H> {
                             called_address,
                         ));
                     }
+                    // If this is a call to the timestamp asserter, extract the function arguments and store them in ValidationTraces.
+                    // These arguments are used by the mempool for transaction filtering. The call data length should be 68 bytes:
+                    // a 4-byte function selector followed by two U256 values.
+                    let address = self.timestamp_asserter_params.as_ref().map(|x| x.address);
+                    if Some(called_address) == address
```

Comment:
I'd write something like`if self.timestamp_asserter_params.map(|x| x.address == called_address).unwrap_or_default()`

Reply:
Actually, to avoid the `.unwrap()` later on, you should do `if let Some(address) = ...` here.